requirements.txt

```diff
@@ -21,13 +21,14 @@ requests
 # modern python dataclasses
 pydantic
 # lock version
-pydantic-yaml==0.11.2
+pydantic_yaml
+
+# for 3.11 - see https://github.com/yaml/pyyaml/issues/724
+PyYAML==5.3.1
```

Comment:
Ah, nice catch this issue with PyYAML has been popping up everywhere it seems!

Reply:
yes